### ISSUE TITLE
Bootstrap USD/BTC liquidity

### ISSUE THREAD

**Issue by ripcurlx** [bisq-network/growth#15]
## Learning Goal/Objective
If we provide enough liquidity for the USD/BTC order book, we bootstrap the market for organic growth.

## Hypothesis
If the experiment is successful, we predict to the total volume to grow by x (everyone placed its own bet) percent.

## Experiment Design
Every participant on the experiment tries to keep two offers (1 buy, 1 sell with low distance from market price) open all the time to provide liquidity.

## Resource Estimation and Probability
Low personal effort and High probability that it will succeed.

## Results
> _- What happened?_
> _- Did we succeed or fail?_
> _- What data did we collect?_
> _- Anything unexpected?_
> _- How close have we been to our prediction?_
> _- Why did we see the result that we did?_

## Learnings
> _-What did we learn by running this experiment?_

## Action Items
> _- Another experiment for this goal?_
> _- How do we progress from here on?_

**Comment by flix1** [comment]
Before we can bootstrap USD liquidity we need to review payment methods and research which USD payment methods can combine security, fraud prevention requirements (no chargebacks) with speed and with a large user base.

**Comment by flix1** [comment]
Localbitcoins has the following payment methods for USD:
-Cash by Mail
-OKPay
-Perfect Money
-Payeer
-Transfers with specific bank
-Paxum
-International Wire (SWIFT)
-WebMoney
-Cash Deposit
-Western Union
-Neteller
-Other online payment: Revolut
-National bank transfer: United States
-Payoneer

**Comment by flix1** [comment]
Market making milestones:

Phase 1: Minimal order book (1 offer on each side, spread < 10%, total offered > 0.1 BTC)
Phase 2: Retail order book (10 offers on each side, spread < 3%, total offered > 5 BTC)
Phase 3: Trading order book (20 offers on each side, spread < 1%, total offered > 10 BTC)
Phase 4: Wholesale order book (50 offers per side, spread < 0.5%, total offered > 50 BTC)
Phase 5: Institutional order book (100 offers per side, spread < 0.25%, total offered > 100 BTC)

**Comment by ripcurlx** [comment]
Was superseded by https://github.com/bisq-network/growth/issues/36.
